README.md

```diff
@@ -29,7 +29,37 @@ Faster for session establishment, not throughput ! SSH3 offers a significantly f
 <i>SSH3 (top) VS SSHv2 (bottom) session establishement with a 100ms ping towards the server.</i>
 </p>
 
-## 🔒 SSH3 is secure
+## ⚡ SSH3 is still experimental
+
+While SSH3 shows promise for faster session establishment, it is 
+still at an early proof-of-concept stage. As with any new complex 
+protocol, **expert cryptographic review over an extended timeframe is required before reasonable security conclusions can be made**.
+
+We are developing SSH3 as an open source project to facilitate community feedback and analysis. However, we **cannot yet endorse its appropriateness for production systems** without further peer review. Please collaborate with us if you have relevant expertise!
+
+## 🥷 Do not deploy the SSH3 server on your production servers for now
+
+Given the current prototype state, we advise *testing SSH3 in sandboxed environments or private networks*. Be aware that making experimental servers directly Internet-accessible could introduce risk before thorough security vetting.
+
+While [hidding](#-your-ssh3-public-server-can-be-hidden) servers behind secret paths has potential benefits, it 
+does not negate the need for rigorous vulnerability analysis before 
+public launch. We are excited by SSH3's future possibilities but 
+encourage additional scrutiny first.
+
+## 🙏 Community support needed
```

Comment:
I would just call it `🙏 Community support` which, to me, looks more like classical community-oriented repos